src/Components/Search/SearchBar.tsx

```diff
@@ -244,7 +245,7 @@ export class SearchBar extends Component<Props, State> {
     ) => ({
       action_type: Schema.ActionType.SelectedItemFromSearch,
       query: state.term,
-      destination_path: href,
+      destination_path: regex.test(href) ? href + "/works-for-sale" : href,
```

Comment:
Hmmm.... what is the value of `href` here? 

Is it `/artist/foo`, `/artwork/bar`, etc? Or is it a full url starting with `http`?

Reply:
🐈 Can use `displayType` prop in place of the regex, that should be "Artist" for all artists in the list.

That propagates from MP here: https://github.com/artsy/metaphysics/blob/e0a2c0e81953571a80eeba405118ca688170864a/src/schema/v2/SearchableItem/index.ts#L42. In the case of artists, that falls thru that helper to here: https://github.com/artsy/metaphysics/blob/e0a2c0e81953571a80eeba405118ca688170864a/src/schema/v2/SearchableItem/SearchableItemPresenter.ts#L93

So, that corresponds to the `label` property of the upstream response.

![Screen Shot 2020-04-07 at 9 22 34 AM](https://user-images.githubusercontent.com/1457859/78674269-56dba080-78b1-11ea-8f55-038cff45dbe4.png)

Looks promising! But what's `label`?

That is in Gravity: https://github.com/artsy/gravity/blob/09ef21978f278cca4184e2bd1db03ade6039712d/app/api/util/search_helpers.rb#L54

The `_type` field comes from our ES index, and corresponds to the `document_type` of a model.

In the version of ES and associated gems we're using, that's defined as: https://github.com/elastic/elasticsearch-rails/blob/5251a5201f0b3fa1278e4366e364f9aaded81be6/elasticsearch-model/lib/elasticsearch/model/naming.rb#L93 (which uses some Rails helpers to get the model name), unless it's overridden (which we do for partner sub-classes): https://github.com/artsy/gravity/blob/09ef21978f278cca4184e2bd1db03ade6039712d/app/models/domain/partner_gallery.rb#L3

I apologize for this insanely pedantic comment. It's just, regexes.... 😅

Reply:
So per @mzikherman's suggestion...

```suggestion
      destination_path: displayType === "Artist" ? `${href}/works-for-sale` : href,
```